docker/build_scripts/build.sh

```diff
@@ -48,11 +48,11 @@ mkdir -p /opt/python
 build_cpythons $CPYTHON_VERSIONS
 rm -rf /usr/local/ssl
 
-# Install patchelf and auditwheel (latest)
-curl -sLO http://nixos.org/releases/patchelf/patchelf-0.9/patchelf-0.9.tar.gz
-tar -xzf patchelf-0.9.tar.gz
-(cd patchelf-0.9 && ./configure && make && make install)
-rm -rf patchelf-0.9.tar.gz patchelf-0.9
+# Install patchelf and auditwheel (latest with unreleased bug fixes)
+curl -sLO http://nipy.bic.berkeley.edu/manylinux/patchelf-0.9njs2.tar.gz
```

Comment:
Indeed if someone finds a way to rootkit our manylinux1 gcc or patchelf binaries the consequences could be dramatic. I think we need to think about hashing all the file of that image and keep an offline copy somewhere and setup a process to audit periodically the public manylinux1 images from quay.io so as to be able to detect compromised binaries.

PyPI is similarly a potential target and there is a draft PEP to devise a more robust security model for it: https://www.python.org/dev/peps/pep-0480/